kolibri/plugins/learn/assets/src/views/CopiesModal.vue

```diff
@@ -39,21 +40,19 @@
 <script>
 
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import genContentLink from '../utils/genContentLink';
+  import useCopies from '../composables/useCopies';
 
   export default {
     name: 'CopiesModal',
     mixins: [commonCoreStrings],
-    props: {
```

Comment:
Let's revert this back to whatever was merged with the Library page refactor PR. Might need to drop a commit or two in the rebase?